inspirehep/records/api/literature.py

```diff
@@ -143,3 +169,141 @@ def _dump_for_es(self):
         serialized_data = super()._dump_for_es()
         serialized_data["_ui_display"] = json.dumps(self.get_ui_data())
         return serialized_data
+
+
+def import_article(identifier: str) -> dict:
```

Comment:
we shouldn't introduce types in this PR, this is a bigger decision to make an affect the rest of the project as well ☺️

Reply:
tbh he's gonna put them in doc string otherwise or remove, perhaps it's not bad to have them like that. It's not like the PR is introducing also a static type checker that is running on CI, so this alone does not seem a big decision to me.

Reply:
Actually I think that it's great that Antonio introduced them. As Harun said, it's not like introducing static checking in CI but it's a good start for at least discussing about them ;)